### PR TITLE
Credential and Environment removed from CB, fetched from Env service

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+ALTER TABLE stack ADD COLUMN IF NOT EXISTS environmentcrn varchar(255);
+ALTER TABLE cluster ADD COLUMN IF NOT EXISTS environmentcrn varchar(255);
+ALTER TABLE datalakeresources ADD COLUMN IF NOT EXISTS environmentcrn varchar(255);
+
+CREATE INDEX idx_stack_envcrn_name ON stack USING btree (environmentcrn, name);
+CREATE INDEX idx_cluster_envcrn_name ON cluster USING btree (environmentcrn, name);
+CREATE INDEX idx_datalakeresources_envcrn_name ON datalakeresources USING btree (environmentcrn, name);
+
+DROP TABLE IF EXISTS environment_network;
```
Comment (bergerdenes) [inline core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql]: no rollback for these, just saying

File: core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
+-- // CB-1570 eliminating environment from core
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack DROP COLUMN IF EXISTS environment_id;
```
Comment (bergerdenes) [inline core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql]: no rollback for this

**Comment by biharitomi** [comment]
@bergerdenes the Environment related APIs will be deleted from the core's API, so it has been leaved in the code temporarily.

File: core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql (hunk 2 above)
Comment (attilapalfi92) [reply core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql]: Added the rollback

File: core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql (hunk 1 above)
Comment (attilapalfi92) [reply core/src/main/resources/schema/app/20190530120120_CB-1570_eliminate_environment_from_core.sql]: added the undo

**Pull request merged**